style.css

```diff
@@ -1,3 +1,436 @@
+*,
+*::before,
+*::after {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  text-decoration: none;
+  list-style: none;
+}
+
+body {
+  width: 375px;
+}
```

Comment:
[OPTIONAL]
So @SAZZAD1Q2 
Your current app's overall styling is a perfect design as per FIGMA design. But I observed this on a different screen size:
![image](https://user-images.githubusercontent.com/61361037/219869998-63cc170b-7563-4731-8d43-49e474a0ff6a.png)

So when I added this 
``` 
body {
    width: 100%;
}
```
that space was removed at all screen sizes. So you can give the full width so that it remove this extra space on the right from all the screen sizes.